### DISCUSSION TITLE
Framework Laptop 13 (13th Gen Intel Core)

### DISCUSSION THREAD

**Discussion by giscus** [framewiki/framewiki.net#30] [Talk]
# Framework Laptop 13 (13th Gen Intel Core)

The unofficial community-driven Framework wiki

https://framewiki.net/framework-laptop-13/13th-gen

**Comment by KellyNyanbinary** [comment]
Today I learn that there are tables. Some data originally in the shared section and there are minor typos. I might fix them later.

**Reply by Morpheus636** [reply]
If you're referring to the cache info, I removed it because it takes up quite a bit of space and is more info than is generally useful to most readers, especially since it's the same between all three CPUs. If someone wants that level of detail, they are more than likely already reading the sources cited.

**Comment by KellyNyanbinary** [comment]
Where did our previous comments go?

**Reply by Morpheus636** [reply]
The comments on citations should've been moved to the contributing page but it looks like they didn't relocate properly. I'm investigating. https://github.com/framewiki/framewiki.net/discussions/29#discussioncomment-7200228

Update: Fixed
